pom.xml

```diff
@@ -121,6 +121,8 @@
     <scala.compat.version>2.10</scala.compat.version>
     <scala.version>2.10.4</scala.version>
     <spark.version>1.3.1</spark.version>
+    <!-- TODO: Remove snapshot dependency when Flink 0.9.1 is released -->
+    <flink.version>0.9-SNAPSHOT</flink.version>
```

Comment:
@alexey : BTW is there such a thing as Flink shell?

On Tue, Oct 20, 2015 at 12:54 AM, Henry Saputra <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/apache/mahout/pull/137#discussion_r42465218>:
>
> > @@ -121,6 +121,8 @@
> >      <scala.compat.version>2.10</scala.compat.version>
> >      <scala.version>2.10.4</scala.version>
> >      <spark.version>1.3.1</spark.version>
> > +    <!-- TODO: Remove snapshot dependency when Flink 0.9.1 is released -->
> > +    <flink.version>0.9-SNAPSHOT</flink.version>
>
> Flink 0.9.1 is out so we could remove the SNAPSHOT label I suppose.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/mahout/pull/137/files#r42465218>.
>